src/backend/gporca/libnaucrates/src/operators/CDXLTableDescr.cpp

```diff
@@ -86,6 +87,26 @@ CDXLTableDescr::MdName() const
 	return m_mdname;
 }
 
+
+// table alias accessors
+const CMDName *
+CDXLTableDescr::MdAlias() const
+{
+	return m_mdalias;
+}
+
+void
+CDXLTableDescr::SetMdAlias(CMDName *alias)
+{
+	if (NULL != m_mdalias)
+	{
+		GPOS_DELETE(m_mdalias);
```

Comment:
instead delete I suggest ensure it is null
```c
GPOS_ASSERT(NULL == m_mdalias);
```

Reply:
Are we going to re-record something? Is this even a valid case?

Reply:
I don't know if it valid or not, but why to prohibit while it costs nothing to support?

Reply:
> why to prohibit while it costs nothing to support?

Then you should provide a test for this branch of code

Reply:
Am I understand correctly that tests and asserts can be considered as mutual exclusive concepts for trivial changes?

Reply:
Something like that.

Reply:
Okay although I am not quite happy with it